TCP_c.go

```
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	conn, _ := net.Dial("tcp", "127.0.0.1:8081")
```

Why do you ignore an error returned by `net.Dial()` ?

---

Do not forget to close connection at the end of function.